internal/storage/postgres/postgres.go

```diff
@@ -493,6 +493,28 @@ func (s pg) GetProfileStats(ctx context.Context, address string) (storage.Stats,
 	return out, nil
 }
 
+func (s pg) GetAllUsersStats(ctx context.Context) (*storage.AllUsersStats, error) {
+	var statsDTO struct {
+		DDV int64   `db:"ddv"`
+		ADV float64 `db:"adv"`
+	}
+
+	if err := sqlx.GetContext(ctx, s.ext, &statsDTO, `
+		WITH pdv AS (
+			SELECT address, sum(updv) - 1000000 as pdv from updv
+			group by address
+		)
+		SELECT SUM(pdv) AS ddv, AVG(pdv) AS adv FROM pdv;
```

Comment:
ADV is average PDV, it should be greater than 1. Seems like ADV = 1000000 + AVG(pdv). Please double check.

Reply:
Ok